src/index.ts

```diff
@@ -53,14 +55,15 @@ export class Marqeta {
   apiAppToken: string
   apiAccessToken: string
   private accessToken: string
-
+  business: BusinessApi
```

Comment:
Typically we have a blank line between the last ivar definition and the constructor.